learn/getting-started/working-with-data/using-query-expressions-to-manipulate-data.md

```diff
@@ -0,0 +1,252 @@
+---
+layout: using-query-expressions-to-manipulate-data
+title: Using Query Expressions to Manipulate Data
+permalink: /learn/using-query-expressions-to-manipulate-data
+description: This simple guide helps you understand how query expressions can be used on data to produce new data 
+active: using-query-expressions-to-manipulate-data
+intro: This simple guide helps you understand how query expressions can be used on data to produce new data 
+redirect_from:
+  - /learn/using-query-expressions-to-manipulate-data
+---
+
+This simple guide helps you understand how query expressions (a.k.a language integrated queries) can be used on data to produce new data.
+
+Ballerina supports first-class support for writing SQL-like queries to process data.
+
+Language-integrated queries can process any Ballerina iterable collection.
```

Comment:
Should we say just `any Ballerina iterable` may be?

Reply:
fixed